src/index.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Russo+One&display=swap');
```

Comment:
- [ ] The mobile version is questionable:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/0e635593-9971-4500-b585-a4d62770ec0e)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/c7598e54-53aa-4153-94a6-0d5d363d4a9a)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/ac3aa867-efba-43ba-854f-ce7c4d1113b4)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/75990edb-756b-4dec-8dae-446d3c09e488)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/34eae0b7-c924-475f-98d1-8cdf9a724441)

In most cases, the content overflows the screen (it does not fit).

Also, in this version, the cars can't be navigated when there are more than 2 cars.

Kindly, touch up your styling to fix these issues, that way, your app will be more usable, and beautiful. 💪